### DISCUSSION TITLE
openwrt 22.03 上用openclash

### DISCUSSION THREAD

**Discussion by jwkinko** [vernesong/OpenClash#2683] [Ideas]
大佬是否会更新在 openwrt 22.03上用

**Comment by wplook** [comment]
拉取dev版本的编译，或者我编译好的https://drive.google.com/file/d/1FwVRjulLY8F-zhxjR9_7PL-DGwI_cwod/view?usp=sharing

**Reply by malikshi** [reply]
Is v 45.57 support nft on openwrt 22.03?
